pkgs/os-specific/linux/i8kutils/default.nix

```
@@ -0,0 +1,34 @@
+{ lib, stdenv, fetchFromGitHub, tcl }:
+
+stdenv.mkDerivation rec {
+  pname = "i8kutils";
+  version = "c993fb1da1bba5c2cd2860c1aa6c3916b4de77e4";
```

use date-based version please, if they don't have releases/tags
```suggestion
  version = "unstable-2021-03-30";
```